.jshintrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Everything looks great here! I hadn't done a separate .jshintrc before but this looks great.